doc/final-design.md

```diff
@@ -49,8 +49,19 @@ Scripts and modules have limited access to the balance transfer functionality vi
         origin: OriginFor<T>,
         bundle: Vec<u8>,
         gas_limit: u64,
-    ) -> DispatchResultWithPostInfo {
+    ) -> DispatchResultWithPostInfo;
 ```
+
+```rust
+    /// Publish a standard library, e.g. Move-Stdlib or Substrate-Stdlib. Sudo user only.
+    #[pallet::call_index(3)]
+    #[pallet::weight(T::WeightInfo::update_stdlib())]
+    pub fn update_stdlib(
```

Comment:
Should we add more information to the function's description or do you think about some protection mechanism, that it is really a bundle?

Reply:
Let's rename it to `update_stdlib_bundle`, and then you don't have to explain anything - the user will know what a bundle is from the previous extrinsic.
And this will leave some space for implementing `update_stdlib_module` in the future